src/http_server.cc

```diff
@@ -1353,8 +1354,24 @@ HTTPAPIServer::HandleRepositoryControl(
             HTTP_RESPOND_IF_ERR(
                 req,
                 param_json.MemberAsString(m.c_str(), &param_str, &param_len));
-            auto param = TRITONSERVER_ParameterNew(
-                m.c_str(), TRITONSERVER_PARAMETER_STRING, param_str);
+
+            TRITONSERVER_Parameter* param = nullptr;
+            if (m == "config") {
+              param = TRITONSERVER_ParameterNew(
+                  m.c_str(), TRITONSERVER_PARAMETER_STRING, param_str);
+            } else if (m.rfind("file:", 0) == 0) {
+              // Decode base64
+              base64_decodestate s;
+              base64_init_decodestate(&s);
+
+              // The decoded can not be larger than the input...
+              binary_files.emplace_back(std::vector<char>(param_len + 1));
```

Comment:
Does a `binary_files` list of vectors need to be maintained here or used later? Seems like just using a temp vector for each iteration would do the same currently.

Reply:
The decoded content is stored in `binary_files` and it needs to be valid until `LoadModelWithParameter` returns